index.html

```diff
@@ -1,269 +1,183 @@
-          <li><a href="#"><img src="./images/medium.png" alt="social"></a></li>
-          <li><a href="#"><img src="./images/github.png" alt="social"></a></li>
-          <li><a href="#"><img src="./images/angellist.png" alt="social"></a></li>
-        </ul>
-      </div>
-    </section>
-    <section class="work-section">
-      <div class="project-card desktop">
-        <div class="mobile-image">
-        <img src="images/photo1.png" alt="First Card">
-      </div>
-      <div class="desktop-image">
-        <img src="images/photo1dt.png" alt="First card-desktop">
-      </div>
-
-
-      <div class="contents-two">
-        <h2 class="head">Tonic</h2>
-        <h4 class="desktop_show">Tonic</h4>
-        <div class="card-detail-box">
-          <ul class="cards-container">
-            <li class="card-detail canopy">CANOPY</li>
-            <li class="card-detail option"><img src="images/bullets.png" alt="bullet" class="circle"> Back End Dev</li>
-            <li class="card-detail option"><img src="images/bullets.png" alt="bullet" class="circle"> 2015</li>
-            <li class="card-detail dcanopy2_face">CANOPY</li>
-            <li class="card-detail optiond2_full"><img src="images/bullets.png" alt="bullet" class="circle"> Back End Dev</li>
-            <li class="card-detail optiond2_full"><img src="images/bullets.png" alt="bullet" class="circle"> 2015</li>
-          </ul>
-
-          <p class="instruction"> A daily selection of privately personalized reads; no accounts or sign-ups required. </p>
-          <p class="instruction-show"> A daily selection of privately personalized reads; no accounts or sign-ups required. </p>
-          
-          <div class="language1 inline">
-            <ul>
-              <li class="langu">html</li>
-              <li class="langu">css</li>
-              <li class="langu">javascript</li>
-              <li class="langu2">html</li>
-              <li class="langu2">css</li>
-              <li class="langu2">javascript</li>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles/style.css" />
+    <script
+      src="https://kit.fontawesome.com/7ec004c2c1.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Anthony's Portfolio</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="header-section">
+        <nav class="navbar">
+          <a href="#" class="logoo">Ezema Anthony</a>
+          <div class="place hide">
+            <span class="close"><img src="img/close.png" alt="#" /></span>
+            <ul class="nav-menu">
```

Comment:
- [ ] Great job so far 👏🏻 However your close.png images is not showing up correctly 👇🏻 

- I think the problem is because you're missing the `./` in your path, let's add `./` path directory before your image URL. It should be like this:  `<span class="close"><img src="./img/close.png" alt="#" /></span>>` (line 28)

 - [ ] Nice implementing the design, however, it would be better if you add a little bit of blur to your overlay in order to match the Figma design and also make a good contrast 🎨 👨🏻‍🎨 

- [OPTIONAL] What about preventing the sub-background from scrolling when the navbar is open? You can use a CSS property `over-flow: hidden;` to do that, which will make your mobile menu correspond to the original design.💯 🥇 

-  Adding these features will make your design look great and look exactly like the Figma design, I highly recommend you do this in the meantime, I'll approve this project since the previous reviewer is skipping this.


| Your design | Figma design |
| --- | --- |
![18](https://user-images.githubusercontent.com/79058364/190303956-b52a760c-d20f-46f6-89f7-b7108554cdb6.png) | ![image](https://user-images.githubusercontent.com/61790587/167505757-7e8c3eba-c166-4f42-842b-829d676cabd5.png) |

Reply:
Thanks all creations made